doc/private_net.rst

```diff
@@ -0,0 +1,103 @@
+Creating a private network and deploying a contract with Remix
+===============
+
+This guide describes creating a private network with a single node and connecting `Remix <https://remix.ethereum.org/>`__ to it through an HTTP RPC interface.
+
+Initializing Aleth in the Command-Line Interface
+------
+
+**1.** Create a new aleth-account.
```

Comment:
* "Create a new aleth-account" -> "Create a new Aleth account"
* I don't think you need the periods in these numbered sections. If there's content after the section title I'd use a colon instead e.g. "Create a new Aleth account:"